plugins/source/gitlab/Makefile

```diff
@@ -0,0 +1,34 @@
+# Test unit
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+# Install tools
+.PHONY: install-tools
+install-tools:
+	@echo Installing tools from tools/tool.go
+	@cat tools/tool.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
+
+.PHONY: lint
+lint:
+	golangci-lint run --config ../../.golangci.yml
+
+.PHONY: gen-code
+gen-code:
+	grep -rl '// Code generated by codegen; DO NOT EDIT.' resources/services/* | xargs rm
+	go run codegen/main.go
+
+.PHONY: gen-mocks
+gen-mocks: install-tools
```

Comment:
Not using `GoMock` any more, `gen-mocks` and `install-tools` targets (as well as `tools/`) can be removed.